components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2096,8 +2098,18 @@ where
         let region_id = derived.get_id();
         meta.set_region(&self.ctx.coprocessor_host, derived, &mut self.fsm.peer);
         self.fsm.peer.post_split();
+        // It's not correct anymore, so set it to None to let split checker update it.
+        self.fsm.peer.approximate_size = None;
```

Comment:
Hmmm, I think we should remove these lines?

Reply:
This is for non-leader peers. As the previous comment says, the size is not correct anymore.

Reply:
If we use the estimated size for non-leader peers but not to update it size immediately, I don't know whether there are 
potential risks?

Reply:
If `self.fsm.peer.approximate_size` is none, the latter `estimated_size` is always none.

Reply:
There is no potential risk because the `approximate_size` and `approximate_keys` are meaningful only on the leader.

Reply:
Should it be meaningful if a follower becomes a leader?

> If self.fsm.peer.approximate_size is none, the latter estimated_size is always none.

Sorry, I make a mistake here.